core/templates/services/assets-backend-api.service.ts

```diff
@@ -212,7 +212,7 @@ export class AssetsBackendApiService {
     }
   }
 
-  private async fetchFile(
+  async fetchFile(
```

Comment:
As this function is called from questions-opportunities-select-difficulty-modal.component.ts hence this cannot be private function.

Reply:
Can we leave this private and use fetchImage or fetchAudio in other places?